packages/stories-utils/src/__tests__/prepareStory.test.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * @jest-environment jsdom
+ */
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import prepareStory from '../prepareStory';
+
+test('no args', () => {
+  const Story = () => <button>brew</button>;
+
+  render(prepareStory(Story));
+  expect(screen.getByRole('button', { name: 'brew' })).toBeTruthy();
+});
+
+test('args', () => {
+  type Args = { children: React.ReactNode };
+  const Story = (args: Args) => <button {...args} />;
+  Story.args = { children: 'hiya' };
+
+  render(prepareStory(Story));
+  expect(screen.getByRole('button', { name: 'hiya' })).toBeTruthy();
```

Comment:
Good idea. That would prevent this possibility:

```js
// when using screen.findByRole (instead of getByRole)
// forget to use async await
expect(screen.findByRole('button')).toBeTruthy();
```

Since `findBy*` queries return a promise, this would always truthy (even when the element isn't found).

So, that's a long way of saying, I like it.

Reply:
I seem to have some cert issues, and can't actually install any new deps through yarn right now...

Will debug that, but punt on this for now.